mmv1/products/cgc/terraform.yaml

```
@@ -72,7 +72,15 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         primary_resource_id: "instance_virtual_display"
         vars:
           instance_virtual_display: "instance-virtual-display"
- 
+    ### OS Login
+      - !ruby/object:Provider::Terraform::Examples
+        name: "os-login"
```

```suggestion
        name: "os_login"
```
this gets inserted as-is into the function name for tests, and hyphens result in errors